src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -273,44 +291,38 @@ export default class ManualBusinessInfo extends Mixins(CountriesProvincesMixin,
 
   // Local properties
   active = false
-  businessLookup = null
   business = {} as ExistingBusinessInfoIF
   jurisdiction = null
   isMrasJurisdiction = false
   jurisdictionErrorMessage = ''
 
-  /** Whether to show this component. */
-  get showComponent (): boolean {
-    return true
-  }
-
-  /** Whether we have a looked-up business. */
-  get haveLookupBusiness (): boolean {
-    return !isEqual(this.business, {})
-  }
-
+  /** Called when this component is mounted. */
   mounted (): void {
-    // get the business info object from the store or initialize it
-    if (this.getExistingBusinessInfo) this.business = this.getExistingBusinessInfo
+    // point business variable to Existing Business Info object from the store, if it exists
+    if (this.getExistingBusinessInfo) {
+      this.business = this.getExistingBusinessInfo
+    } else {
+      this.business = cloneDeep(EmptyExistingBusinessInfoIF)
+      this.setExistingBusinessInfo(this.business)
```

Comment:
This resets the local object variable and then syncs it to the store state. As with the other component, the local variable and the store state point to the same object in memory so we don't need to continually update the store when the user enters some input.